specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -660,6 +660,46 @@
         }
       }
     },
+    "EnvironmentSkuProperties": {
+      "description": "Managed Environment resource SKU properties.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the Sku.",
+          "externalDocs": {
+            "url": "https://aka.ms/container-apps-pricing"
+          },
+          "enum": [
+            "Consumption",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Consumption",
+                "description": "Consumption SKU of Managed Environment."
+              },
+              {
+                "value": "Premium",
+                "description": "Premium SKU of Managed Environment."
+              }
+            ]
+          }
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Capacity of the SKU (number of deployed nodes of the SKU). For Consumption SKU capacity must be specified as 0."
+        }
+      },
+      "required": [
+        "name",
+        "capacity"
```

Comment:
If must be 0, could we make it optional?
And if user select premium, then create 2 app, one is MemoryOptimized and the other is ComputeOptiomized, then what we will do? set both node pool to this capacity?

Reply:
Removed `capacity` property. This was inspired from [arm sku api contract](https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/Doc.aspx?sourcedoc=%7bD68FD846-9EC2-4D20-925D-133626DD6D1B%7d&file=arm-sku-api.docx&action=default&mobileredirect=true&share=IQFG2I_Wwp4gTZJdEzYm3W0bATGtnXbTsaqMOdZ4ErrL-Jo&cid=b5d55c0f-c7f2-4739-8940-2a9c3193d8fb). Will bring it back if needed